front/app/components/admin/ContentBuilder/Widgets/ButtonMultiloc/index.tsx

```diff
@@ -74,8 +74,9 @@ const Button = ({ text, url, type, alignment }: ButtonProps) => {
           id="e2e-button"
           width={alignment === 'fullWidth' ? '100%' : 'auto'}
           buttonStyle={type}
-          text={localize(text)}
-        />
+        >
+          <span style={{ whiteSpace: 'normal' }}>{localize(text)}</span>
+        </ButtonComponent>
       )}
     </Box>
```

Comment:
I looked into doing this for our component library Button and general Button component as well, but it breaks a lot of our current button UI and causes wrapping when we don't actually want it to.

The best solution for new feels like just wrapping the text for Content Builder buttons so they wrap when we want them to. Open to other suggestions though :)

Reply:
Does it also break if you make the change in Button in the UI folder?

Reply:
Yeah it does unfortunately.

Reply:
You can also use the Text component.